Split/SplitClient.swift

```diff
@@ -119,6 +118,11 @@ public final class SplitClient: NSObject, SplitClientProtocol {
         impression.treatment = treatment
         impression.time = Date().unixTimestamp()
         ImpressionManager.shared.appendImpressions(impression: impression, splitName: splitName)
+        
+        if let externalImpressionHandler = config?.getImpressionListener() {
+            impression.attributes = attributes
```

Comment:
Are we sure we are not storing on the appendImpressions the attributes? Maybe creating a new impression?

Reply:
Yes, we're sure. Impression class was changed to struct. That way impression is passed by value, not by ref so that attributes are not passed to addImpressions function.